index.bs

```diff
@@ -480,36 +480,16 @@ starts at 0 the first time {{getCurrentFrameData()}} is invoked for a given
 {{VRDisplay}}.
 
 <dfn attribute for="VRFrameData">leftProjectionMatrix</dfn>
-A 4x4 matrix describing the projection to be used for the left eye's rendering,
-represented as a 16 element array in column major order. This value may be
-passed directly to WebGL's uniformMatrix4fv function. It is highly recommended
-that applications use this matrix without modification. Failure to use this
-projection matrix when rendering may cause the presented frame to be distorted
-or badly aligned, resulting in varying degrees of user discomfort.
+A 4x4 matrix describing the projection to be used for the left eye's rendering, given as a 16 element array in column major order. This value may be passed directly to WebGL's uniformMatrix4fv function. It is highly recommended that applications use this matrix without modification. Failure to use this projection matrix when rendering may cause the presented frame to be distorted or badly aligned, resulting in varying degrees of user discomfort.
```

Comment:
I'm not sure I follow. We're not describing a matrix math libraries, and when passing into WebGL it's implied that it expects OpenGL style matrices. As such the storage convention seems like the only relevant thing to specify here. Or am I missing something?